ci/gpu/build.sh

```diff
@@ -77,13 +79,13 @@ gpuci_conda_retry build -c ${LIBCUCIM_BLD_PATH} -c conda-forge -c rapidsai-night
 
 # Install cuCIM and its dependencies
 gpuci_logger "Installing cuCIM and its dependencies"
-gpuci_conda_retry install -y -c ${LIBCUCIM_BLD_PATH} -c ${CUCIM_BLD_PATH} \
-    "rapids-build-env=$MINOR_VERSION.*" \
```

Comment:
I found that there is no rapids-build-env package whose version is `$MINOR_VERSION.*` as we switched to CalVer.

https://anaconda.org/rapidsai/rapids-build-env/files

Reply:
@gigony, this will need to be re-added. We use the `rapidsai-nightly` conda channel for pre-release packaging. You can find the cal-ver packages here: https://anaconda.org/rapidsai-nightly/rapids-build-env/files

Reply:
Somewhat related. I think we are missing an `activate` line (this is probably my fault). Was trying to fix that here ( https://github.com/rapidsai/cucim/pull/45/commits/495aa7316661b2efc95957d39c49f8831fb73f91 ). Think we may need to add that as well